.github/workflows/rust.yml

```diff
@@ -17,6 +17,28 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Build
+      working-directory: ./tests
```

Comment:
brauchst hier nicht, cargo test macht das build auch

Reply:
mir passt aber ehrlich gesagt nicht, dass Du bei Dir im Workspace einige Crates ignorest nur damit man auf github Toplevel bauen kann. Ich fänd es aus Nutzersicht schöner, wenn ich mit cargo build im Repo Root alles bekomme. Da pflege ich lieber den CI Build ein bisschen mehr.

Reply:
dachte wir haben es besprochen, dass wir es so machen bis wir ne Loesung fuer SDL haben.. mir ist tatsaechlich auch lieber im root cargo build mache und sehe dass alles lauft

Reply:
hmm, also wenn man den Workspace richtig aufsetzt geht das glaub ich schon. theoretisch kannst du für ein Crate eine .cargo Datei anlegen, mit der du festlegst mit welcher Toolchain das Crate zu bauen ist, d.h. wir hätten 3 Crates, die Executables erzeugen:
1. Unittesttarget, gebaut für x64
2. Windowstgarget, gebaut für x64, braucht SDL2
3. Cortex M0 Target gebaut mit thumbv6m-none-eabi
Dadurch, dass man in den Cargo.tomls der drei Crates direkt angibt, welche von den sonstigen Crates verwendet werden, werden diese so referenzierten Crates dann auch mit der Toolchain des Hauptcrates gebaut.

Reply:
nebenbei hatten wir nur besprochen, dass wir das Windowstarget dann nicht auf github bauen, um das zu erreichen gibt es ja mehrere Wege :)

Reply:
es ist zwar richtig, dass man in .cargo ein config mit dem entprechenen Target legt ... problem was ich gesehen habe ist dass wenn du in root cargo build machst .. ignoriert es komplett

Reply:
und crasht 
![image](https://user-images.githubusercontent.com/10464534/109390885-80ac1480-7914-11eb-8ed0-b2d8eee7711d.png)